src/CryptoAxus.API/Controllers/ArtistController.cs

```diff
@@ -262,6 +309,37 @@ private IReadOnlyList<Links> CreateArtistLinks(string? id = null, int? userId =
                                        $"{HttpContext?.Request.Scheme}://{HttpContext?.Request.Host}{Constants.ApiValue}");
         links.Add(link);
 
+        link = new Links(Url.RouteUrl("GetOffersReceivedByArtist", new { userId }),
+                         "get_offersRecivedByArtist",
+                         Constants.GetMethod);
+        link.Href = link.Href?.Replace(Constants.ApiValue,
+                                       $"{HttpContext?.Request.Scheme}://{HttpContext?.Request.Host}{Constants.ApiValue}");
+        links.Add(link);
+        return links.AsReadOnly();
+    }
+
+    private IReadOnlyList<Links> CreateOfferLinks( int? userId = null, string? fields = "", string? id = null)
+    {
+        Links link;
+        List<Links> links = new List<Links>();
+        if (!string.IsNullOrWhiteSpace(fields))
+        {
+            link = new Links(Url.RouteUrl("GetArtistOffersReceivedByArtist", new { userId }),
+                             Constants.SelfRel,
+                             Constants.GetMethod);
+            link.Href = link.Href?.Replace(Constants.ApiValue,
+                                           $"{HttpContext?.Request.Scheme}://{HttpContext?.Request.Host}{Constants.ApiValue}");
+            links.Add(link);
+        }
+        else
+        {
+            link = new Links(Url.RouteUrl("GetArtistOffersReceivedByArtist", new { userId }),
+                             Constants.SelfRel,
+                             Constants.GetMethod);
+            link.Href = link.Href?.Replace(Constants.ApiValue,
+                                           $"{HttpContext?.Request.Scheme}://{HttpContext?.Request.Host}{Constants.ApiValue}");
+            links.Add(link);
+        }
         return links.AsReadOnly();
     }
```

Comment:
Wrong implementation. Should be related to the offers that are returned in the response.

Reply:
Resolved